extensions/wikia/AdEngine/js/config/desktop.js

```diff
@@ -42,8 +42,8 @@ define('ext.wikia.adEngine.config.desktop', [
 		log(slotName, 5, logGroup);
 		log(['getProvider', 'tracking opted ' + (isTrackingOptedIn ? 'in' : 'out')], log.levels.info, logGroup);
 
-		// If wgShowAds set to false, hide slots
-		if (!context.opts.showAds) {
+		// If wgShowAds set to false or the browser is a steam one, hide slots
+		if (!context.opts.showAds || context.opts.isSteamBrowser) {
```

Comment:
What about bidders. Should we disable them too?

Reply:
It won't handle Featured Video too

Reply:
I will create an improvement task for bidders, featured video handled in the second commit